SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -556,11 +529,13 @@ FString GetComponentSetNameBySchemaType(ESchemaComponentType SchemaType)
 		return SpatialConstants::DATA_COMPONENT_SET_NAME;
 	case SCHEMA_OwnerOnly:
 		return SpatialConstants::OWNER_ONLY_COMPONENT_SET_NAME;
-	case SCHEMA_Handover:
+	case SCHEMA_ServerOnly:
 		return SpatialConstants::HANDOVER_COMPONENT_SET_NAME;
```

Comment:
HANDOVER_COMPONENT_SET_NAME

Reply:
Changed to SERVERONLY_COMPONENT_SET_NAME now